app/code/Magento/Customer/Block/Account/AuthenticationPopup.php

```diff
@@ -105,14 +105,23 @@ public function getBaseUrl()
 
     /**
      * Get customer register url
-     *
+     * @deprecated
```

Comment:
Please add deprecation comment due to requirements

Reply:
@Den4ik , deprecation comment added.